Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -184,34 +318,56 @@ void HiresTexture::Prefetch()
       }
       if (iter != s_textureCache.end())
       {
-        for (const Level& l : iter->second->m_levels)
+        for (const Level& tex : iter->second->m_levels)
```

Comment:
Actually I asked about that and they told me that the `if (base_filename.find("_mip") == std::string::npos)` line in which the data size iterator is sitting it, means everything else except the _mip files. I'm not sure yet what it does if there is a _mip, if it even counts those.

Otherwise this wouldn't work, since there's no _mip files in the Hypatia's Texture Pack for Zelda WW.